code/datums/status_effects/debuffs.dm

```diff
@@ -287,6 +288,85 @@
 	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
 	..()
 
+/datum/status_effect/eldritch
+	duration = 15 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	var/mutable_appearance/marked_underlay
+	var/effect_sprite = ""
+
+/datum/status_effect/eldritch/on_apply()
+	if(owner.mob_size >= MOB_SIZE_HUMAN)
+		marked_underlay = mutable_appearance('icons/effects/effects.dmi', effect_sprite)
+		marked_underlay.pixel_x = -owner.pixel_x
+		marked_underlay.pixel_y = -owner.pixel_y
+		owner.underlays += marked_underlay
+		return TRUE
+	return FALSE
+
+
+/**
+  * What happens when this mark gets popped
+  *
+  * Adds actual functionality to each mark
+  */
+/datum/status_effect/eldritch/proc/on_effect()
+	qdel(src) //what happens when this is procced.
+
+/datum/status_effect/eldritch/Destroy()
+	if(owner)
+		owner.underlays -= marked_underlay
+	QDEL_NULL(marked_underlay)
+	return ..()
+
+/datum/status_effect/eldritch/be_replaced()
+	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
+	return ..()
+
+//Each mark has diffrent effects when it is destroyed that combine with the mansus grasp effect.
+/datum/status_effect/eldritch/flesh
+	id = "flesh_mark"
+	effect_sprite = "emark1"
+
+/datum/status_effect/eldritch/flesh/on_effect()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.bleed_rate += 5
+		H.blood_volume -= 200
+	return ..()
+
+/datum/status_effect/eldritch/ash
+	id = "ash_mark"
+	effect_sprite = "emark2"
+	///Dictates how much damage and stamina loss this mark will cause.
+	var/repetitions = 1
+
+/datum/status_effect/eldritch/ash/on_creation(mob/living/new_owner, _repetition = 1)
+	. = ..()
+	repetitions = _repetition
+
+/datum/status_effect/eldritch/ash/on_effect()
+	if(iscarbon(owner))
+		var/mob/living/carbon/carbon_owner = owner
+		carbon_owner.adjustStaminaLoss(10 * repetitions)
+		carbon_owner.adjustFireLoss(5 * repetitions)
```

Comment:
Each repetition would already double the damage done if there was no scaling.

Scaling damage can be pretty troublesome, and I'm not seeing any checks here for infinite loops.

Do instead diminishing damage on jumps, start higher if necessary, and keep a list of the affected mobs to avoid loops.

Reply:
aight

Reply:
done